java-kmsinventory/README.md

```diff
@@ -0,0 +1,201 @@
+# Google KMS Inventory API Client for Java
+
+Java idiomatic client for [KMS Inventory API][product-docs].
+
+[![Maven][maven-version-image]][maven-version-link]
+![Stability][stability-image]
+
+- [Product Documentation][product-docs]
+- [Client Library Documentation][javadocs]
+
+> Note: This client is a work-in-progress, and may occasionally
+> make backwards-incompatible changes.
+
+
+## Quickstart
+
+
+If you are using Maven, add this to your pom.xml file:
+
+
+```xml
+<dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-kmsinventory</artifactId>
+  <version>0.0.0</version>
```

Comment:
We need to manually add release please set up for this version

Reply:
Added in [afa8738](https://github.com/googleapis/google-cloud-java/pull/9162/commits/afa8738719c29695f2dba266fbea0c3d52d8f0d1), following https://github.com/googleapis/google-cloud-java/pull/9150 (cc/ @ddixit14)

Reply:
Thanks! Can you please add it to our new module creation guide as well? cc @suztomo